### ISSUE TITLE
[bug] `<rh-alert>`  toast animation should be improved

### ISSUE THREAD

**Issue by bennypowers** [RedHat-UX/red-hat-design-system#1778]
### Describe the bug

The animation for `RhAlert.toast` doesn't exactly align with the description provided in the docs

### Steps to reproduce

1. call `RhAlert.toast({ message: 'hi' })`

### Expected behaviour

animation is as described in docs

### Element HTML

```HTML
n/a
```

### Operating System (OS)

iOS, macOS, Linux, Windows

**Comment by markcaron** [comment]
@bennypowers what needs to be done to align with the documented animation? Are you able to pick this up?

**Comment by bennypowers** [comment]
well there used to be an animation section in the rh-alert style or guideline docs but it's not there anymore

If the existing animation is acceptable to @coreyvickery and @marionnegp, then this can be closed.

**Comment by coreyvickery** [comment]
@bennypowers Where can I see a sample? Only the Toast variant should be animated.

**Comment by bennypowers** [comment]
I dont think we have a specific demo for that but you can see an example by using any of the copy buttons e.g. on the tokens pages

**Comment by bennypowers** [comment]
@coreyvickery FYI I added a button to the toast demo which creates a new toast

the main branch is failing to deploy to github pages, but you can see it on this deploy preview branch

https://deploy-preview-1992--red-hat-design-system.netlify.app/elements/alert/demo/toast/

**Comment by coreyvickery** [comment]
@bennypowers Animation and persistence time is good, thank you.

**Comment by markcaron** [comment]
Closing since #1992 is merged and @coreyvickery approved.
